app/src/main/res/values/strings.xml

```diff
@@ -127,19 +127,14 @@
     </string>
     <string name="about_background_heading">Background</string>
     <string name="about_background_body_text">
-        We\'ve all been there: It's one of those group situations, where, after a little while, the
-        group branches off into pairs. Jenny, the runner, goes
-        with Fred, also a runner. James always watches indie flicks. He\'ll end up with April
-        because she does the same.\n
+        We've all been there: messages that go nowhere; likes, winks, and pokes sent with no response; 
+        conversations that fizzle and die. How can it be so hard to meet someone you like and who actually likes you back?\n
         \n
-        When we get to a big  city with a big dating pool, we suddenly see the creme of the crop.
-        Some of us won\'t settle for anything less than chasing unattainable "gods" and
-        "goddesses." Beeline slots you with only a small number of matches that you actually have
-        a chance with.\n
+        Fortunately, Beeline's got your back. It slots you with only a small number of matches that are 
```

Comment:
"Fortunately, Beeline's got your back." is a great line to start expressing the actual difference.

The strength of Beeline is NOT that it matches with a small number of matches. There are other sites that match with a small number of matches. People are frustrated with them because the matches suck, and people aren't able to find other matches on their own in the app.

The difference is that Beeline knows through the comparisons exactly how much you are attracted to each person. Beeline also knows exactly who of your top choices is highly attracted back at you. Beeline then matches you only with the people that have the highest mutual attraction. So, these are the people that you have the best chance of starting something. (You and he/she expressed high attraction for the match. No fallible/weak/faulty algorithm made the decision.)

That was very verbose to clearly explain it. The copy should follow the short conversational tone that you started really well with "We've all been there... Fortunately, Beeline's got your back..."